Sources/Client/State/LoadingState.swift

```diff
@@ -1,9 +1,10 @@
 import Foundation
-import DeltaCore
+
+// MARK: - LoadingState
+
 
 enum LoadingState {
-  case loading
+  case none
```

Comment:
What is the none case for? If it's just renaming loading i think loading was better. It's probably not necessary to have a none case. Why is there LoadingState and StartupState, it seems a bit confusing having two states that sound like they mean similar things

Reply:
I see now, I think you meant to delete LoadingState

Reply:
Nope, the old loading state is now called StartupState and the LoadingState being used right now can be changed from anywhere inside the app to display / dismiss a loading screen
Changed it like this as it wasn't possible to do it before: the old LoadingState (now StartupState) needed resources only available during startup to revert / update to the .done state